rules/Proj0503.md

```diff
@@ -0,0 +1,54 @@
+---
+parent: Licensing
+ancestor: Rules
+---
+
+# Proj0503: Package license is unknown
+Using a [NuGet](https://www.nuget.org) package implies that you
+and/or your company explicitly agree with the legally binding conditions of the
+license and the copyright of the owner of the package.
+
+If a third-party package license comes with a custom license (other then one
+specified by the [SPDX](https://spdx.dev)), this rule ensures that the license
+must have been aproved.
+
+To aprove a license, a `<<ThirdPartyLicense>` node has to be added with a
+matching include and a hash of the license file. That hash is communicated for
+licenses which lack such a registration.
+
+## Non-compliant
+``` xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="SonarAnalyzer.CSharp" Version="10.6.0.109712" />
+  </ItemGroup>
+
+</Project>
+```
+
+## Compliant
+``` xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="SonarAnalyzer.CSharp" Version="10.6.0.109712" />
+  </ItemGroup>
+
+  <ItemGroup Label="Aproved licenses">
+    <ThirdPartyLicense Include="SonarAnalyzer.CSharp" Hash="ZOAgZmx18wSWq5KpOpWd2bB9123" />
+  </ItemGroup>
+
+</Project>
+```
+
+This rule can detect used licenses in NuGet spec files, but can not be
```

Comment:
I would probably highlight the second bit a bit more.

This rule helps in having a developer or company decide on whether to accept a license.
It is not responsible for a project's licenses, and should not be seen as legal advice.
This documentation is also not legal advice.